database-commons/src/main/java/io/cdap/plugin/db/source/AbstractDBSource.java

```diff
@@ -168,12 +168,12 @@ public Schema getSchema() throws SQLException {
   }
 
   private Schema loadSchemaFromDB(Connection connection, String query) throws SQLException {
-    Statement statement = connection.createStatement();
+    Statement statement = RetryUtils.createStatementWithRetry((RetryPolicy<Statement>) retryPolicy, connection,
```

Comment:
can we use try with resources here?

Reply:
Yes, added try-with-resource. 037a0d4fc724a108d41966303ef96f14940b661d